app/views/browse/persistent-volume-claim.html

```diff
@@ -18,6 +18,17 @@ <h1 class="contains-actions">
               <li ng-if="!pvc.spec.volumeName">
                 <a ng-href="{{pvc | editYamlURL}}" role="button">Edit YAML</a>
               </li>
+              <li>
+                <edit-pvc
+                  ng-if="(pvcVersion | canI : 'update') && isExpansionAllowed"
```

Comment:
nit: I'd flip the order of this check so it can short circuit more efficiently since `isExpansionAllowed` is already calculated

```
ng-if="isExpansionAllowed && (pvcVersion | canI : 'update')"
```